libraries/botbuilder-dialogs-adaptive-teams/.gitignore

```diff
@@ -0,0 +1,6 @@
+/**/node_modules
+/**/.vscode
+/**/lib/*
```

Comment:
The any-directory pattern is not needed 😄 

This file is missing the `_ts3.4` folder, can you update this .gitignore with the contents of the botbuilder-azure-blobs [.gitignore](https://github.com/microsoft/botbuilder-js/blob/eeb2e36b6d9470b45f33f617cd1e10705270d5af/libraries/botbuilder-azure-blobs/.gitignore)?

Reply:
What are we doing w/ package-lock.json? I see the other libraries don't have them in their .gitignore. Are we just being careful about not committing them?

Reply:
No, it’s likely a global rule at the root.

(I’m not actually sure why we have these library level gitignores)

Reply:
We use yarn, so `yarn.lock` in the repo's root should be in source control. No other lock files should be checked in unless they are for packages that aren't a part of the workspace (i.e., teams scenario packages, functional tests, etc.).